src/components/calcite-date-picker-month-header/calcite-date-picker-month-header.tsx

```diff
@@ -134,6 +134,7 @@ export class CalciteDatePickerMonthHeader {
           </CalciteHeading>
           <span class="year-wrap">
             <input
+              aria-label="year"
```

Comment:
@Elijbet this needs a `intlYear` prop so that a user can localize and set this value.

Reply:
Ok, I'll take care of that.